ui/src/components/EntryDetailed.tsx

```diff
@@ -62,6 +62,7 @@ const EntrySummary: React.FC<any> = ({data}) => {
 };
 
 export const EntryDetailed: React.FC<EntryDetailedProps> = ({entryData}) => {
+    console.log("entryData.rulesMatched:", entryData.rulesMatched)
```

Comment:
I guess this is a debug log that should be removed?
@mertyildiran

Reply:
@gadotroee I'm sorry, I forgot to remove that.